com.unity.render-pipelines.universal/Runtime/Passes/DrawSkyboxPass.cs

```diff
@@ -68,7 +71,26 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
             else
 #endif
             {
-                context.DrawSkybox(renderingData.cameraData.camera);
+                if((DebugHandler != null) && DebugHandler.IsActiveForCamera(ref renderingData.cameraData))
```

Comment:
What about the XR codepath?

Reply:
Yep, looks like this won't work with XR right now. If the "DebugRenderSetup" stuff were encapsulated, this pass would benefit from it as well.

Reply:
The logic was actually simpler here - we just skip skybox rendering altogether for certain modes (overdraw & wireframe modes), so I moved the code to the top and it now applies to XR as well.